lib/bundle.js

```diff
@@ -14,7 +14,7 @@ const _ = require('lodash'),
     jsonPointerDecodeAndReplace,
     generateObjectName
   } = require('./jsonPointer'),
-  traverseUtility = require('traverse'),
+  traverseUtility = require('neotraverse'),
```

Comment:
Should be neotraverse/legacy for CJS usage. Same in other places.

Reply:
Fixed, thanks!